src/API/GBFS/StationStatus.php

```diff
@@ -34,12 +35,52 @@ class StationStatus extends BaseRoute {
 	public function prepare_item_for_response( $item, $request ): stdClass {
 		$preparedItem                      = new stdClass();
 		$preparedItem->station_id          = $item->ID . "";
-		$preparedItem->num_bikes_available = count( Item::getByLocation( $item->ID ) ); // TODO should be the item availability in this moment
+		$preparedItem->num_bikes_available = $this->getItemCountAtLocation($item->ID);
 		$preparedItem->is_installed        = true;
 		$preparedItem->is_renting          = true;
```

Comment:
Hier `is_installed` oder `is_renting` in Zukunft vielleicht sowas wie getTimeframeCount, also ob überhaupt items verfügbar sind.

Reply:
Ich habe mich dafür mit Ortwin ausgetauscht (der kennt sich ganz gut mit GBFS aus) und er meinte, dass das grundsätzlich nur beschreibt ob eine Station installiert ist und theoretisch Räder beherbergen könnte. Er meinte, dass das bei uns mehr Sinn ergeben würde das einfach immer auf true zu lassen. CommonsBooking passt da nicht so perfekt in die GBFS Logik

Reply:
Allgemein passt GBFS nicht ganz, dast stimmt und im speziellen für `is_installed` stimmt die Aussage ebenso, da geht es ja laut Spezifikation eher um Ausleihen von Items die z.B. am Bürgersteig "installiert" sind. 

Für `is_renting` steht in der Spezifikation aber 

```
Is the station currently renting vehicles?

true - Station is renting vehicles. Even if the station is empty, if it would otherwise allow rentals, this value MUST be true.
false - Station is not renting vehicles.

If the station is temporarily taken out of service and not allowing rentals, this field MUST be set to false.

If a station becomes inaccessible to users due to road construction or other factors this field SHOULD be set to false. Field SHOULD be set to false during hours or days when the system is not offering vehicles for rent.
```
https://gbfs.org/specification/reference/#station_statusjson

Ich würde das für CommonsBooking folgendermaßen übersetzen:

* true, wenn die Location mindestens einen Zeitrahmen mit einem Item zugeordnet hat (unabhängig von laufenden Buchungen)
* false, wenn 
  * die Location gar keinen Zeitrahmen zugordnet hat 
  * oder entweder 
    * alle Zeitrahmen deaktiviert sind 
    * oder alle Zeitrahmen für den Zeitpunkt bzw. Range der Anfrage eine Restriction zugeordnet haben.

Ich würde das aber auch nicht in diesen Pull-Request packen.